using_images/db_images/postgresql.adoc

```diff
@@ -174,46 +182,75 @@ method to change passwords for the database user (`*POSTGRESQL_USER*`) and
 *postgres* administrator user is by changing the environment variables
 `*POSTGRESQL_PASSWORD*` and `*POSTGRESQL_ADMIN_PASSWORD*`, respectively.
 
+You can view the current passwords by viewing the pod in the web console or by
+listing the environment variables:
+
+----
+$ oc env pod/<pod_name> --list
+----
+
 Changing database passwords through SQL statements or any way other than through
 the environment variables aforementioned will cause a mismatch between the
 values stored in the variables and the actual passwords. Whenever a database
 container starts, it resets the passwords to the values stored in the
 environment variables.
 
-You can change these passwords with:
+To change these passwords, update one or both of the desired environment
+variables for the related deployment configuration(s) using the `oc env`
+command. If multiple deployment configurations utilize these environment
+variables, for example in the case of an application created from a template,
+you must update the variables on each deployment configuration so that the
+passwords are in sync everywhere. This can be done all in the same command:
 
 ----
-$ oc env dc postgresql \
+$ oc env dc <dc_name> [<dc_name_2> ...] \
   POSTGRESQL_PASSWORD=<new_password> \
   POSTGRESQL_ADMIN_PASSWORD=<new_admin_password>
```

Comment:
@rhcarvalho I tried testing this out first using the django-psql-example template. After updating POSTGRESQL_PASSWORD and POSTGRESQL_ADMIN_PASSWORD, the postgresql pod redeployed fine, but my django-psql-example pod kept failing to deploy, with the error:

```
FATAL:  password authentication failed for user "django"
```

I noticed that that particular template has a DATABASE_PASSWORD variable as well (which looks to be getting set for POSTGRESQL_PASSWORD?):

https://github.com/openshift/django-ex/blob/master/openshift/templates/django-postgresql.json#L264-L265

So I updated all three (POSTGRESQL_PASSWORD, POSTGRESQL_ADMIN_PASSWORD, and DATABASE_PASSWORD), and then everything redeployed fine.

The other postgresl template (rails-postgresql-example) did not have this issue (no DATABASE_PASSWORD). So should I add a caveat about DATABASE_PASSWORD to the topic here, or is this something that needs addressing in the django-psql-example template itself?